src/pages/Profile/subpages/Settings/Traveler.scss

```diff
@@ -3,12 +3,27 @@
   flex-direction: column;
   margin: 8px;
   .traveler-settings-header {
-    color: #071529;
+    position: relative;
+    color: #a1a9b7;
     font-size: 24px;
     font-weight: 550;
     padding-bottom: 4px;
     border-bottom: 1px solid rgba(248, 248, 252, 0.2);
     margin-bottom: 20px;
+    .save-button {
```

Comment:
The style of the button doesn't match the site. We should keep the button designs similar. Also don't forget `cursor: pointer`;